cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseDDLExecutorFactory.java

```diff
@@ -88,4 +90,9 @@ protected String getHBase10CHD550ClassName() {
   protected String getHBase12CHD570ClassName() {
     return "co.cask.cdap.data2.util.hbase.DefaultHBase12CDH570DDLExecutor";
   }
+
+  @Override
+  protected CConfiguration getCConfiguration() {
```

Comment:
Don't add a field to a class just for the sake of returning it to someone else, unless for POJO class, which this class is not